src/applications/appeals/10182/components/ContestableIssuesWidget.jsx

```diff
@@ -8,7 +8,8 @@ import set from 'platform/utilities/data/set';
 import { setData } from 'platform/forms-system/src/js/actions';
 
 import { IssueCard } from './IssueCard';
-import { SELECTED, MAX_LENGTH, LAST_ISSUE, REVIEW_ISSUES } from '../constants';
+import { SELECTED, MAX_LENGTH, REVIEW_ISSUES } from '../constants';
+import { LAST_ISSUE } from '../../shared/constants';
```

Comment:
I think everything except `REVIEW_ISSUES` should be pulled from the shared constants